index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio page</title>
+    <link rel="stylesheet" href="style.css"> 
+</head>
+<body>
+    <header class="container">
+        <nav class="nav">
+        <a class="my-logo" href="index.html">
+        <img src="assets/images/My%20Logo.svg" alt="Portofolio logo"> 
+        </a>
+        <a class="hamburger" href="#">
+        <img src="assets/images/burger.svg" alt="Hambugrer_menu">   
+        </a>
+    </nav>
+    </header>
+    <section class="main">
+      <h1 class="header">I'am Sentayhu <br>  Glad to see <br>you!</h1>
+      <p class="intro">
+          I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<a class="lets-connect" href="#">LETS CONNECT
+</a>
+<br>
+<div class="connect-via">
+    <img  src="assets//images/twitter.png" alt="twitter">
+   <div class="linkdin">
+        <img src="assets//images/l%20in%20linkdin.svg" alt="linkdin">
+        <img src="assets//images/n%20in%20linkdin.svg" alt="linkdin">   
+</div>
+     <img src="assets//images/medium.svg" alt="medium">
+     <img src="assets//images/github.svg" alt="github"> 
+     <img src="assets//images/angellist.svg" alt="angellist">
+</div>   
```

Comment:
- [ ] As this is a list of elements, I suggest you use an unordered list `<ul>` tag, with each element being a list item that includes a link to your social media accounts.